build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -229,25 +229,32 @@ class SeleniumWebDriverController {
       height,
     } = rect;
 
-    // Calculate the window borders, so we can set the correct size to get the
-    // desired content size.
-    const results = await Promise.all([
-      this.driver.manage().window().getRect(),
-      this.driver.findElement(By.tagName('html')),
-    ]);
-    // No Array destructuring allowed?
-    const winRect = results[0];
-    const htmlElement = results[1];
+
+    await this.driver.manage().window().setRect({
+      width,
+      height,
+    });
+
+    // Check to make sure we resized the content to the correct size.
+    const htmlElement = this.driver.findElement(By.tagName('html'));
     const htmlElementSizes = await Promise.all([
       htmlElement.getAttribute('clientWidth'),
```

Comment:
should this be `getRect`? instead of `getAttribute`?

Reply:
I tried using `getRect`, but it seems like the height does not line up with what I expect. Not sure why though.